plugins/roadie-backstage-entity-validator/src/validator.ts

```diff
@@ -50,7 +51,7 @@ const VALIDATORS = {
   product: ajvCompiledJsonSchemaValidator(productSchema),
 };
 
-function modifyPlaceholders(obj) {
+function modifyPlaceholders(obj: any) {
```

Comment:
I know this isnt your code. Maybe we should do a pass on the syntax here to use more modern syntax.